### PR TITLE
Fixed potential formula

### PR DESCRIPTION
**Pull request by hidekb**
Fixed the formula of potential and force.

### CODE REVIEW

**Comment by govarguz** [comment]
LGTM, @stuehn what do you think? 
Mind also updating the (c)...

File: src/interaction/VSpherePair.hpp [hunk 1]
```diff
@@ -80,8 +80,8 @@ namespace espressopp {
       virtual ~VSpherePair() {};
 
       void preset() {
-      	mfh = -(3.0/2.0);
-    	mth = -(5.0/2.0);
+      	mth = -(3.0/2.0);
+    	mfh = -(5.0/2.0);
         ef1 = epsilon*pow(1.0L*(2*M_PIl/3.0), 1.0L*mth);
```
Comment (niktre) [inline src/interaction/VSpherePair.hpp]: What is the point to ask for `long double` in the power and multiply it by `real`?

**Comment by niktre** [comment]
btw, is it the first example of using doxygen for dev docs?

**Comment by junghans** [comment]
@hidekb : Is this still related to the original pull-request?

**Comment by govarguz** [comment]
@junghans and @hidekb it is certainly not, although the first one had only a minimal correction requested was not merged

**Comment by hidekb** [comment]
@junghans please reject latest ones.

**Comment by junghans** [comment]
Please resubmit the first commit as a new pull request!

**Pull request closed without merging**